scripts/run_on_popular_crates/src/main.rs

```diff
@@ -0,0 +1,100 @@
+use std::time::Duration;
+use std::collections::HashMap;
+use std::process::Command;
+use std::io::Write;
+use crates_io_api::{CratesQuery, Sort};
+
+/// To get the rustdoc of the baseline, we first create a placeholder project somewhere
+/// with the baseline as a dependency, and run `cargo rustdoc` on it.
+fn create_rustdoc_manifest_for_crate_version(
+    name: &str,
+    version: &str,
+    features: &HashMap<String, Vec<String>>
+) -> cargo_toml::Manifest<()> {
+    use cargo_toml::*;
+
+    Manifest::<()> {
+        package: {
+            let mut package = Package::new(name, version);
+            package.publish = Inheritable::Set(Publish::Flag(false));
+            Some(package)
+        },
+        workspace: Some(Workspace::<()>::default()),
+        lib: {
+            let product = Product {
+                path: Some("lib.rs".to_string()),
+                ..Product::default()
+            };
+            Some(product)
+        },
+        dependencies: {
+            let project_with_features = DependencyDetail {
+                version: Some(version.to_string()),
+                features: features
+                    .iter()
+                    .map(|(key, _values)| key.clone())
```

Comment:
Is this not just `.keys().cloned()`?

Reply:
Done in 4f463225ca84523b8d04f140ce3ffec64a8524e8.